iati/guidance_and_support/zendeskhelper.py

```diff
@@ -0,0 +1,17 @@
+def generate_ticket(request):
+    """Generate a JSON formatted support ticket for Zendesk given a request object"""
+    path = request.path
+    captcha = request.POST['phone'] == ""
```

Comment:
Could we call this `is_captcha_valid` ??
I was a bit confused looking at this line at first but then worked out what was going on. This var name might have helped.

Reply:
Plus potentially add an inline comment `# Set to False if the honeypot has been entered` ??